zuul.d/projects.yaml

```diff
@@ -276,6 +276,11 @@
     templates:
       - noop-jobs
 
+- project:
+    name: github.com/ansible-security/ids_install
+    templates:
+      - noop-jobs
+
```

Comment:
so, we need to split this over 2 commits. The reason being, we cannot add jobs to zuul, until we import the project into tenant config below.  So, remove this for now, we can merge this, then add this back.

Reply:
@pabelanger do I also need to remove that from `github/project.yaml`?

Reply:
no, you can keep that. we just need to remove it from here first.